src/Microsoft.DotNet.Configurer/xlf/LocalizableStrings.it.xlf

```diff
@@ -36,6 +32,15 @@ Configurazione...
         <target state="translated">Non è possibile inizializzare la cache NuGet. {0} non riuscito. Errore: {1}</target>
         <note />
       </trans-unit>
+      <trans-unit id="NugetCachePrimeMessage">
+        <source>Configuring...
+-------------------
+A command is running to initially populate your local package cache, to improve restore speed and enable offline access. This command will take up to a minute to complete and will only happen once.</source>
+        <target state="new">Configuring...
+-------------------
+A command is running to initially populate your local package cache, to improve restore speed and enable offline access. This command will take up to a minute to complete and will only happen once.</target>
+        <note />
+      </trans-unit>
```

Comment:
I was wondering how this slipped in, but I guess it was because of my change and the corresponding change to this text being merged together without giving ci a chance to catch it.